website/static/js/filepage/index.js

```diff
@@ -17,6 +18,65 @@ var Panel = utils.Panel;
 
 var EDITORS = {'text': FileEditor};
 
+var MakeClient = function(element) {
+    makeClient(element);
+};
+
+var CopyButton = {
+    view: function(ctrl, params) {
+        return m('span.input-group-btn', m('button#copyBtn.btn.btn-default.btn-md[type="button"][style="height:' + params.copyButtonHeight + '"][data-clipboard-text="' + params.link + '"]',
```

Comment:
`copyButtonHeight` is too unnecessarily verbose; `height` will do.

For readability, you can pass the `style` attribute as an object:

```js
m('span..', {style: {height: params.height})
```